File: app/src/main/java/com/example/james/rms/Core/ServePath/UserProfileServePath.java
package com.example.james.rms.Core.ServePath;

import com.example.james.rms.NetWork.ServeProfile;

/**
 * Created by Jamie on 16/4/2017.
 */

public class UserProfileServePath {

    public static String serve_checkLogin(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getLogin_checklogin();
        String serve_path = serve+path;
        return serve_path;
    }

    public static String serve_findAll(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getLogin_findAll();
        String serve_path = serve+path;
        return serve_path;
    }

    public static String serve_findByPartyId(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getLogin_findByPartyId();
        String serve_path = serve+path;
        return serve_path;
    }

    public static String serve_findbyFacebookId(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getLogin_findByFacebookId();
        String serve_path = serve+path;
        return serve_path;
    }

    public static String serve_save(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getLogin_save();
        String serve_path = serve+path;
        return serve_path;
    }


}

File: app/src/main/java/com/example/james/rms/Core/ServePath/UserProfileServePath.kt
package com.example.james.rms.Core.ServePath

import com.example.james.rms.NetWork.ServeProfile

/**
 * Created by Jamie on 16/4/2017.
 */

object UserProfileServePath {
    val serve = ServeProfile.serve

    fun serve_checkLogin(): String {
        val path = ServeProfile.login_checklogin
        return serve + path
    }

    fun serve_findAll(): String {
        val path = ServeProfile.login_findAll
        return serve + path
    }

    fun serve_findByPartyId(): String {
        val path = ServeProfile.login_findByPartyId
        return serve + path
    }

    fun serve_findbyFacebookId(): String {
        val path = ServeProfile.login_findByFacebookId
        return serve + path
    }

    fun serve_save(): String {
        val path = ServeProfile.login_save
        return serve + path
    }

}
